### PR TITLE
return json response from API

### PR DESCRIPTION
**Pull request by shiranr**
Move error handling into handlers.
return json from API

### CODE REVIEW

**Review by balteravishay — APPROVED** [review]
nice!

File: functional-tests/tests/test_anonymizer.py [hunk 1]
```diff
@@ -24,7 +24,8 @@ def test_given_anonymize_called_with_valid_request_then_expected_valid_response_
 
     response_status, response_content = anonymize(request_body)
 
-    expected_response = "hello world, my name is ANONYMIZED. My number is: 03445****"
+    expected_response = '{"text":"hello world, my name is ANONYMIZED. ' \
```
Comment (navalev) [inline functional-tests/tests/test_anonymizer.py:27]: are'nt we using "response" and not "text"?

File: presidio-anonymizer/app.py [hunk 2]
```diff
@@ -57,6 +49,17 @@ def anonymizers() -> Tuple[str, int]:
                 )
                 return ErrorResponse(e.args[0]).to_json(), 500
```
Comment (navalev) [inline presidio-anonymizer/app.py:50]: why not add self.app.errorhandler also here?

**Review by omri374 — APPROVED** [review]

File: functional-tests/tests/test_anonymizer.py (hunk 1 above)
Comment (shiranr) [reply functional-tests/tests/test_anonymizer.py:27]: Correct, will be fixed.

**Pull request merged**